scripts/flattener.py

```diff
@@ -1164,7 +1162,14 @@ def main(mfinal_id):
 	global cxg_adata_raw
 	global cxg_obs
 	mfinal_obj = lattice.get_object(mfinal_id, connection)
-	logging.basicConfig(filename= mfinal_id + '_outfile_flattener.log', filemode='w', level=logging.INFO)
+
+	# Checking for presence / creating output folder and associated sub-folders
+	if os.path.exists(output_dir) == False:
+		os.mkdir(output_dir)
+	if os.path.exists(output_dir + '/' + fltnr_dir) == False:
+		os.mkdir(output_dir + '/' + fltnr_dir)
+
+	logging.basicConfig(filename= output_dir + '/' + fltnr_dir + '/' + mfinal_id + '_outfile_flattener.log', filemode='w', level=logging.INFO)
```

Comment:
For this filename, since there are so many variable, might be more readable as:
`logging.basicConfig(filename="{}/{}/{}_outfile_flattener.log".format(output_dir, fltnr_dir, mfinal_id), filemode='w', level=logging.INFO)`